src/services/fileStorage/proxy-service.js

```diff
@@ -362,20 +374,24 @@ const signedUrlService = {
 			});
 	},
 
-	async patch(_id, data, params) {
+	async patch(id, data, params) {
 		const { payload } = params;
 		const { userId } = payload;
 		const strategy = createCorrectStrategy(payload.fileStorageType);
-		const fileObject = await FileModel.findOne({ _id }).exec();
+		const fileObject = await FileModel.findOne({ _id: id }).exec();
 
 		if (!fileObject) {
 			throw new NotFound('File seems not to be there.');
 		}
 
 		const creatorId = fileObject.permissions[0].refPermModel !== 'user' ? userId : fileObject.permissions[0].refId;
 
-		return canRead(userId, _id)
-			.then(() => strategy.getSignedUrl({ userId: creatorId, flatFileName: fileObject.storageFileName, action: 'putObject' }))
+		return canRead(userId, id)
```

Comment:
should be named canWrite() for that situation

Reply:
Correct.